src/main/java/uk/gov/hmcts/reform/iacaseapi/domain/handlers/presubmit/RequestFeeRemissionAipPreparer.java

```diff
@@ -107,9 +93,11 @@ public PreSubmitCallbackResponse<AsylumCase> handle(
                     asylumCase.write(PREVIOUS_REMISSION_DETAILS, remissionDetailsAppender.getRemissions());
                     assignLateRemissionValuesToRemissionValues(asylumCase);
                     clearPreviousAndLateRemissionFields(asylumCase);
+                    asylumCase.write(IS_LATE_REMISSION_REQUEST, YesOrNo.YES);
```

Comment:
just for clarity. can you move the write above the assignLateRemissionValuesToRemissionValues function?, same below.